index.js

```diff
@@ -8,5 +8,13 @@ module.exports = {
     return {
       fastboot: require('./lib/commands/fastboot')
     };
+  },
+
+  contentFor: function(type, config) {
+    // When using a dist built html file, we need to reinsert 
+    // content-for body so fastboot can insert its content.
+    if (type === 'body') {
+      return "{{content-for 'body'}}";
```

Comment:
Can you tweak this and the [associated replacement function](https://github.com/tildeio/ember-cli-fastboot/blob/038a71dc3d64901bfb14e5d637a57651f69ce895/lib/models/server.js#L24-L31) to use an inline comment instead of `{{content-for 'body'}}`?

Maybe something like:

```javascript
if (type === 'body') {
  return "// EMBER_CLI_FASTBOOT_BODY";
}
```

Then tweak the replacement to look for that string instead?

---

I find that *much* easier to grok (and doesn't overload the concept of `{{content-for 'body'}}` like ember-cli-fastboot does today).